src/array-repeat-strategy.js

```diff
@@ -109,11 +109,11 @@ export class ArrayRepeatStrategy {
         let {index, removed, addedCount} = splices[i];
         mergeSplice(repeat.__queuedSplices, index, removed, addedCount);
       }
-      repeat.__array = array.slice(0);
+      repeat.__array = array.slice(0, array.length);
```

Comment:
@Gheoan it seems is the best option, and also, 0 and array.lenght are the default values for slice, so you can safely remove them